### ISSUE TITLE
"tar: unrecognized option '--include'" on first run

### ISSUE THREAD

**Issue by tmarplatt** [raylee/tldr#27]
Hello, I just ran tldr for the first time on a fresh Mint 19 install. Here's the output:

```tmarplatt@tmarplatt:~$ tldr
tar: unrecognized option '--include'
Try 'tar --help' or 'tar --usage' for more information.
USAGE: tldr [options] <command>

[options]
    -l, --list:      show all available pages
    -p, --platform:  show page from specific platform rather than autodetecting
    -u, --update:    update, force retrieving latest copies of locally cached files
    -c, --cache:     update local cache of all tldr commands
    -h, -?, --help:  this help overview

<command>
    Show examples for this command

The client caches a copy of all pages and the index locally under
/home/tmarplatt/.config/tldr. By default, the cached copies will expire in 14 days.
```

Indeed, `tar --usage` lists no `--include` option:

```
Usage: tar [-AcdrtuxGnSkUWOmpsMBiajJzZhPlRvwo?] [-C DIR] [-T FILE] [-X FILE]
            [-g FILE] [-f ARCHIVE] [-F NAME] [-L NUMBER] [-b BLOCKS]
            [-H FORMAT] [-V TEXT] [-I PROG] [-K MEMBER-NAME] [-N DATE-OR-FILE]
            [--add-file=FILE] [--directory=DIR] [--exclude=PATTERN]
            [--exclude-backups] [--exclude-caches] [--exclude-caches-all]
            [--exclude-caches-under] [--exclude-ignore=FILE]
            [--exclude-ignore-recursive=FILE] [--exclude-tag=FILE]
            [--exclude-tag-all=FILE] [--exclude-tag-under=FILE] [--exclude-vcs]
            [--exclude-vcs-ignores] [--no-null] [--no-recursion] [--no-unquote]
            [--no-verbatim-files-from] [--null] [--recursion]
            [--files-from=FILE] [--unquote] [--verbatim-files-from]
            [--exclude-from=FILE] [--anchored] [--ignore-case] [--no-anchored]
            [--no-ignore-case] [--no-wildcards] [--no-wildcards-match-slash]
            [--wildcards] [--wildcards-match-slash] [--catenate]
            [--concatenate] [--create] [--diff] [--compare] [--delete]
            [--append] [--list] [--test-label] [--update] [--extract] [--get]
            [--check-device] [--listed-incremental=FILE] [--incremental]
            [--hole-detection=TYPE] [--ignore-failed-read] [--level=NUMBER]
            [--seek] [--no-check-device] [--no-seek] [--occurrence[=NUMBER]]
            [--sparse-version=MAJOR[.MINOR]] [--sparse] [--keep-old-files]
            [--keep-directory-symlink] [--keep-newer-files]
            [--no-overwrite-dir] [--one-top-level[=DIR]] [--overwrite]
            [--overwrite-dir] [--recursive-unlink] [--remove-files]
            [--skip-old-files] [--unlink-first] [--verify]
            [--ignore-command-error] [--no-ignore-command-error] [--to-stdout]
            [--to-command=COMMAND] [--atime-preserve[=METHOD]] [--clamp-mtime]
            [--delay-directory-restore] [--group=NAME] [--group-map=FILE]
            [--mode=CHANGES] [--mtime=DATE-OR-FILE] [--touch]
            [--no-delay-directory-restore] [--no-same-owner]
            [--no-same-permissions] [--numeric-owner] [--owner=NAME]
            [--owner-map=FILE] [--preserve-permissions] [--same-permissions]
            [--same-owner] [--preserve-order] [--same-order] [--sort=ORDER]
            [--acls] [--no-acls] [--no-selinux] [--no-xattrs] [--selinux]
            [--xattrs] [--xattrs-exclude=MASK] [--xattrs-include=MASK]
            [--file=ARCHIVE] [--force-local] [--info-script=NAME]
            [--new-volume-script=NAME] [--tape-length=NUMBER] [--multi-volume]
            [--rmt-command=COMMAND] [--rsh-command=COMMAND] [--volno-file=FILE]
            [--blocking-factor=BLOCKS] [--read-full-records] [--ignore-zeros]
            [--record-size=NUMBER] [--format=FORMAT] [--old-archive]
            [--portability]
            [--pax-option=keyword[[:]=value][,keyword[[:]=value]]...] [--posix]
            [--label=TEXT] [--auto-compress] [--use-compress-program=PROG]
            [--bzip2] [--xz] [--lzip] [--lzma] [--lzop] [--no-auto-compress]
            [--gzip] [--gunzip] [--ungzip] [--compress] [--uncompress]
            [--backup[=CONTROL]] [--dereference] [--hard-dereference]
            [--starting-file=MEMBER-NAME] [--newer-mtime=DATE]
            [--newer=DATE-OR-FILE] [--after-date=DATE-OR-FILE]
            [--one-file-system] [--absolute-names] [--suffix=STRING]
            [--strip-components=NUMBER] [--transform=EXPRESSION]
            [--xform=EXPRESSION] [--checkpoint[=NUMBER]]
            [--checkpoint-action=ACTION] [--full-time] [--index-file=FILE]
            [--check-links] [--no-quote-chars=STRING] [--quote-chars=STRING]
            [--quoting-style=STYLE] [--block-number] [--show-defaults]
            [--show-omitted-dirs] [--show-snapshot-field-ranges]
            [--show-transformed-names] [--show-stored-names]
            [--totals[=SIGNAL]] [--utc] [--verbose] [--warning=KEYWORD]
            [--interactive] [--confirmation] [--help] [--restrict] [--usage]
            [--version] [FILE]...
```

tldr seems to be working just fine though. Thanks!

**Comment by raylee** [comment]
Thanks for listing the distro (and your report!). I'll spin up a Mint 19 install in a VM and see if the option has been renamed in that version of tar. In the meantime you can remove the '--include' lines from the tar invocation in the script to avoid the error, the only effect will be that extra unnecessary files are extracted from the tldr master file.

**Comment by raylee** [comment]
That was an embarrassing mistake on my part. I just pushed a commit to fix this for both BSD and GNU versions of tar. Thanks again for the report!
